keymaster/Android.bp

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright (C) 2024 The Android Open-Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+cc_binary {
+    name: "android.hardware.keymaster@3.0-service.optee",
+    relative_install_path: "hw",
+    init_rc: ["3.0/android.hardware.keymaster@3.0-service.optee.rc"],
+    vintf_fragments: ["3.0/android.hardware.keymaster@3.0-service.optee.xml"],
+    vendor: true,
+    cflags: [
+        "-DANDROID_BUILD",
+        "-Wall",
+        "-Wextra",
```

Comment:
Would it make sense to add the additional flags as done for the `wait_for_keymaster_optee` below?

Reply:
Humm maybe but it should be done another patch.

The intention of this commit is to do a pure conversion of Android.mk to Android.bp without changing the current parameters (shared libs, cflags ...)